apps/library/static/admin/play.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function(jQuery) {
+    jQuery(function($) {
+        let cityField = $('#play_form > div > fieldset.module.aligned > div.form-row.field-city');
+        let yearField = $('#play_form > div > fieldset.module.aligned > div.form-row.field-year');
```

Comment:
Можно упростить и ускорить без потери качества
```suggestion
        let cityField = $('#play_form > div.form-row.field-city');
        let yearField = $('#play_form > div.form-row.field-year');
```

Reply:
Почитал jquery standart, очень даже интересно https://lab.abhinayrathore.com/jquery-standards/https://lab.abhinayrathore.com/jquery-standards/

Reply:
У меня не сработало) тут, кстати, играет роль куки (или кэши?) - я такие вещи тестирую после очистки в браузере.

Reply:
Тут оказывается, нужно указать все вложенности. 
` $('#play_form >>> div.form-row.field-city');`
Пусть остаётся как есть